File: Algo/hr_implement/1/catAndMouse.py
"""
You are given queries in the form of x, y, z and representing the respective positions for cats A and B  and X for mouse . 
Complete the function to return the appropriate answer to each query, which will be printed on a new line.
"""

def catAndMouse(x, y, z):
	diff_x_z = abs(x - z )
	diff_y_z = abs(y - z)

	if diff_x_z == diff_y_z:
		return 'Mouse C'
	elif diff_x_z < diff_y_z:
		return 'Cat A'
	else:
		return 'Cat B'


if __name__ == "__main__":
	print catAndMouse(1,2,3)

File: Algo/hr_implement/1/catAndMouse.js
function catAndMouse(x,y,z) {
    let diff_x_z = Math.abs(x - z),
    diff_y_z =  Math.abs(y-z);

    if(diff_x_z === diff_y_z) {
        return 'Mouse C';
    } else if(diff_x_z < diff_y_z) {
        return 'Cat A';
    } else {
        return 'Cat B';
    }

}

console.log(catAndMouse(1,2,3));